__tests__/components/ProfileTasks.test.js

```diff
@@ -14,11 +14,15 @@ describe('ProfileTasks', () => {
           title: 'Province of residence',
           icon: 'circle',
           link: '/dashboard',
+          areaLabel: 'taks1',
```

Comment:
Just nitpicking but should be `task1` :yum:

Reply:
good catch, typo fixed